### DISCUSSION TITLE
提供一个可用音源

### DISCUSSION THREAD

**Discussion by ikun0014** [lyswhut/lx-music-desktop#2094] [General]
[URL](https://lxmusic.ikunshare.com/script)
目前暂不限制请求速率
可搭配我改的版本下企鹅和网易的AI母带
有能力推荐前往[LX Music Api Server](https://github.com/MeoProject/lx-music-api-server)项目自行搭建

**Comment by sanguanxi** [comment]
谢谢爹

**Comment by 248385** [comment]
ikun大佬 用不了了
